SimPEG/EM/NSEM/SurveyNSEM.py

```diff
@@ -64,27 +83,31 @@ def eval(self, f):
     def evalDeriv(self, f):
         raise Exception('Use Sources to project fields deriv.')
 
-#################
+#########
 # Data
-#################
+#########
 
 
-class Data(SimPEGsurvey.Data):
-    '''
+class Data(SimPEGsurvey.Data, DataNSEM_plot_functions):
```

Comment:
The way much of the em code handles these types of things is in the init file, eg for FDEM
```
import SrcFDEM as Src
import RxFDEM as Rx
```
then you get `EM.FDEM.Src`, but naming (and chasing errors) is still fairly transparent.